### DISCUSSION TITLE
希望增加一个移动消息弹幕通知案例

### DISCUSSION THREAD

**Discussion by DiXuanAoYi** [WPFDevelopersOrg/WPFDevelopers#9] [Ideas]
希望增加一个移动消息弹幕通知案例，并着重优化弹幕性能和用户体验（异步）
弹幕从右往左移动，碰到屏幕边缘就自动消失

**Comment by yanjinhuagood** [comment]
已经增加Sample。

**Comment by yanjinhuagood** [comment]
也可以看看SkiaSharp自绘弹幕 [地址](https://github.com/yanjinhuagood/SkiaSharpBarrage)
